backend/src/graphql/resolver/UserResolver.test.ts

```diff
@@ -919,6 +923,7 @@ describe('UserResolver', () => {
   describe('updateUserInfos', () => {
     describe('unauthenticated', () => {
       it('throws an error', async () => {
+        jest.clearAllMocks()
```

Comment:
What's the reason to clearAllMocks here?
There is no logger.test after the testcase.

Reply:
same